z2/docs/deposit.md

```diff
@@ -48,8 +54,9 @@ $ echo '{"secret_key":"96252e38af375be21d9eb30a6b88abc3836acecaeb2240731fb42e029
   z2 deposit --chain zq2-prototestnet \
   --peer-id  12D3KooWGu8PBoj6vMPafnhA2P7sLumSV1NhQJZ2W2AGiBgc5ATW \
   --private-key 96252e38af375be21d9eb30a6b88abc3836acecaeb2240731fb42e0299e14419 \
-  --reward-address 0xe29a3e99a6997B1571DA24d6517e7b3acaFB5d9e \
+  --reward-address 0x3946f9872247af2eb4fe44c81c463e801925b8d4 \
```

Comment:
We should use the original reward address `0xe29a3e99a6997B1571DA24d6517e7b3acaFB5d9e` in the example, it's not supposed to be the same as the signing address

Reply:
My bad you did mention this - must have gotten lost in the threads earlier. Updated